app/src/main/res/layout-large-land/activity_login.xml

```diff
@@ -188,12 +188,13 @@
 
                     <Button
                         android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
                         android:layout_gravity="center"
                         android:layout_margin="@dimen/padding_normal"
-                        android:theme="@style/PrimaryFlatButton"
-                        android:id="@+id/become_member"
+                        android:background="@color/md_grey_500"
                         android:text="Become a member"
-                        android:layout_height="wrap_content" />
+                        android:textColor="@color/md_white_1000"
+                        android:theme="@style/PrimaryFlatButton" />
```

Comment:
Would be better if you create a button style in styles.xml and apply that style in all the button. 
It helps to eliminate redundancy in xml. You can do that in different PR .

Reply:
New Fix https://github.com/open-learning-exchange/myplanet/pull/812